src/pages/organizations.js

```diff
@@ -88,7 +88,17 @@ export const query = graphql`
     topOrganizations: allAirtable(
       filter: {
         table: { eq: "Organizations" }
-        data: { Categories: { elemMatch: { id: { eq: $categoryId } } } }
+        data: {
+          Role: {
+            in: [
+              "Products & Services"
+              "Research & Development"
+              "Advocacy"
+              "Network"
+            ]
+          }
```

Comment:
It might be nice to use a separate airtable view, to keep this logic on the backend.

Reply:
@jasonLaster I agree with your assessment here but it turns out not to be any easier to query this in Airtable. The rule we want is basically "Any org that has on or more Roles not equal to 'Capital'". I think we can roll with this for now.